src/auth/handler.ts

```diff
@@ -1 +1,10 @@
-export const authMeHandler = async () => ({ data: { username: 'akhilmhdh' } });
+import { FastifyReply, FastifyRequest } from 'fastify';
+
+export interface AuthIinterface {
+  authMeHandler: any;
+}
+export class AuthHandler implements AuthIinterface {
+  authMeHandler(_req: FastifyRequest, reply: FastifyReply) {
```

Comment:
```reply.send```

What's this? A fastify implemented that is server. Keep only bussiness logic here. This should take only required this like the object it needs or parameters it needs and return result. The reply send should be done from the other side